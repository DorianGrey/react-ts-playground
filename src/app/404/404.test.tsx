import * as React from "react";
import * as ReactDOM from "react-dom";
import NotFound from "./404";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<NotFound />, div);
});
