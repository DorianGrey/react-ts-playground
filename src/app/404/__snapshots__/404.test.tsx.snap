// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default rendering test renders content as intended 1`] = `
<div>
  <div>
    <h4
      class="MuiTypography-root makeStyles-centered-1 MuiTypography-h4"
    >
      404 | Well, That about Wraps It Up for This Page
    </h4>
    <blockquote>
      <p>
        ’I refuse to prove that I exist,’ says Page, ’for proof denies faith, and without faith I am nothing.’
      </p>
      <p>
        ’But,’ says Man, ’The Babel fish is a dead giveaway, isn’t it? It could not have evolved by chance. It proves you exist, and so therefore, by your own arguments, you don’t. QED.’
      </p>
      <p>
        ’Oh dear,’ says Page, ’I hadn’t thought of that,’ and promptly disappears in a puff of logic.
      </p>
      <p
        class="makeStyles-thanksText-2"
      >
        – Thanks to
         
        <a
          class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
          href="https://www.elastic.co/de/whatever"
          rel="noopener"
          target="_blank"
        >
          https://www.elastic.co
        </a>
         
        for this story
      </p>
    </blockquote>
  </div>
</div>
`;
